Source/Constraint.swift

```diff
@@ -362,7 +362,7 @@ public class Constraint {
         // special logic for updating
         if updateExisting {
             // get existing constraints for this view
-            let existingLayoutConstraints = reverse(layoutFrom!.snp_installedLayoutConstraints)
+            let existingLayoutConstraints = Set<LayoutConstraint>(layoutFrom!.snp_installedLayoutConstraints)
```

Comment:
This is just cosmetic and it's for simplification purposes. This avoids the next for-loop.